Source/Core/Core/Boot/Boot.h

```diff
@@ -42,7 +43,8 @@ class CBoot
                           std::string* title_id = nullptr);
 
 private:
-  static bool DVDRead(u64 dvd_offset, u32 output_address, u32 length, bool decrypt);
+  static bool DVDRead(const DiscIO::IVolume& volume, u64 dvd_offset, u32 output_address, u32 length,
```

Comment:
The reason this one is different is because the other ones can take `nullptr` to mean that there is no disc, whereas DVDRead doesn't accept `nullptr` because its entire point is that it accesses the volume. The visual noise is intentional. By forcing callers to use `*`, it becomes easier to see that DVDRead must not be called with `nullptr`, which forces some of the calling code to have `nullptr` checks. I don't want those `nullptr` checks to be inside DVDRead, because then DVDRead sometimes would do nothing at all despite there being no clear indication of that in the calling code.